### ISSUE TITLE
Assassin Mode and UU Accelerator are not translated into English

### ISSUE THREAD

**Issue by alfky12** [gnuton/asuswrt-merlin.ng#685]
**Router Model Affected**
Models: TUF-AX3000 (v1)

**Firmware Version Affected**
TUF-AX3000_3004_388.8_2-gnuton1_puresqubi.w
TUF-AX3000_3004_388.8_2-gnuton1_tuf_puresqubi.w

**Is this bug present in upstream Merlin releases too?**
Don't know

**Describe the bug**
After finally being able to enable Assassin Mode on TUF-AX3000 non-China region, it turns out that Assassin Mode and UU Accelerator labels are not translated into English, as seen in the screenshot. Please add appropriate translations.

**Expected behavior**
1. 刺客模式 = Assassin Mode
2. 网易UU加速器 = UU Accelerator
3. UU Accelerator page is also translated into English

**Screenshots**
Not translated to english:
![Screenshot (200)](https://github.com/user-attachments/assets/b04e7692-cf78-47b8-9a5c-4d883cec05a7)

UU Accelerator page:
![Screenshot (201)](https://github.com/user-attachments/assets/98f716c6-d84c-40a4-bb8a-872a57682ad5)

BTW, 4T4R activated on non-china region TUF-AX3000:
![Screenshot (199)](https://github.com/user-attachments/assets/7585b322-7795-4400-88a3-0f155b31e0b1)

**Comment by Jacky10X** [comment]
@alfky12 Hello, I am sorry to bother you.
Could you teach us how to enable 4T4R?
Is it something related to cfe? or firmware?

**Comment by Jacky10X** [comment]
It seems your VHT MCS is not fully unlocked?
All should be 0-11 instead of 0-9.

**Comment by alfky12** [comment]
> It seems your VHT MCS is not fully unlocked? All should be 0-11 instead of 0-9.

That's because I previously set the Modulation Scheme up to MCS 9.
If it is set up to MCS 11, the result will be like this:

![Screenshot (205)](https://github.com/user-attachments/assets/3cf34f38-a1f6-4e89-b056-1245696047d0)

**Comment by gnuton** [comment]
@alfky12 can you describe step by step how did you manage to get this working (posssibly without using CFE?)
I may think to enable this by default

**Comment by Jacky10X** [comment]
My method is using [this](https://github.com/JasonJTucker/asus_nvram_editing) to enable 4T4R. (Change it to CN)
but with limited options.
Some firmware authors said it is the limit of the firmware. (The firmware only accepts CN from cfe to enable 4T4R)
I think the second method is to modify the firmware (with cfe?) to unlock the 4T4R for all regions/modes.

**Comment by alfky12** [comment]
> @alfky12 can you describe step by step how did you manage to get this working (posssibly without using CFE?) I may think to enable this by default

@gnuton  Yes I edited the CFE, but, as far as I know, the CFE/bootloader is now separated from the factory nvram on new generation asus routers. So it can be said that what I edited was the factory nvram, not the CFE/bootloader that is on the mtd0 partition.
So this method is safer than editing the CFE/bootloader.

Modifications at the firmware level should be possible, as the SWRT dev did. I have compared, the factory nvram is not changed by flashing that firmware.

**Comment by alfky12** [comment]
Hi @gnuton please remember this issue.
Can’t find for translating Assassin Mode.
But I found this for translating uu accelerator:

![Image](https://github.com/user-attachments/assets/6268f92a-d059-4bd3-902d-0b8a66a67702)

![Image](https://github.com/user-attachments/assets/94be0abd-bcc2-4fde-8c3b-7e3055a054aa)

**Comment by gnuton** [comment]
Hi!
Feel free to send me the translation as patch
